CodeStarterCamp_Week1/main.swift

```diff
@@ -13,73 +13,53 @@ func drawPeperoBody(body: String, topping: String) {
     print(topping + body + topping)
 }
 
-func drawPeperoBar(lenght: Int, topping: String, body: String) {
+func drawPeperoBodyAndLength(length: Int, topping: String, body: String) {
     if body == "***" {
-        for _ in 1...lenght {
+        for _ in 1...length {
             drawPeperoBody(body: body, topping: topping)
         }
     }
     else {
-        for i in 1...lenght {
-            if (i % 2) != 0{
+        for oddNumber in 1...length {
+            if (oddNumber % 2) != 0 {
                 print(" | |")
-            }
-            else {
+            } else {
                 print(body)
             }
         }
     }
 }
 
-func makePepero(lenght: Int, topping: String, body: String, barLength: Int, bar: String) {
-    drawPeperoBody(body: body, topping: topping)
-    drawPeperoBar(lenght: lenght, topping: topping, body: body)
-    drawPeperoBarLength(BarLength: barLength)
+func makePepero(lenght: Int, topping: String, body: String, sickLength: Int, bar: String) {
+    drawPeperoBodyAndLength(length: lenght, topping: topping, body: body)
+    drawPeperoBarLength(sickLength: sickLength)
 }
 
-// 실행 예시 1
-print("<정보>")
-print("길이: 10")
-print("몸통: ***")
-print("토핑:")
-print("막대길이: 4")
 
-makePepero(lenght: 10, topping: " ", body: "***", barLength: 4, bar: " | |")
+func printPeperoData (lenght: Int, body: String, topping: String, sickLength: Int){
```

Comment:
```suggestion
func printPeperoData (lenght: Int, body: String, topping: String, sickLength: Int) {
```
함수 선언의 괄호 앞에서 한 칸 띄워주시는 게 좋습니다 🙂